### PR TITLE
Feature/modal design

### PR DESCRIPTION
**Pull request by ewal**
Updated design according to style guide.

-  Closing icon in header not changed. Needs more investigation to see if it is possible to change the icon in a non painful way.
- Cancel button not changed to the light version. Will implement the new button in another branch.

---

- New background color in header
- Thinner surrounding shadow
- New class "footnotes" for adding just that.
- Darker border around the modal
- Lighter border between body content and header

---

screen shot 2015-03-02 at 15 47 38

### CODE REVIEW

File: src/mnd-bootstrap/javascript/_modal.scss [hunk 1]
```diff
@@ -14,18 +14,43 @@ category: JavaScript
...
+}
+
+.modal-header {
+  background-color: $gray-lightest;
+}
+
+.modal-dialog {
+  .footnote {
+    color: $gray-light;
+    font-size: 12px;
```
Comment (jorkas) [inline src/mnd-bootstrap/javascript/_modal.scss]: @ewal change this to the $font-size-small variable instead!

Comment (ewal) [inline src/mnd-bootstrap/javascript/_modal.scss]: Thanks!

**Comment by jorkas** [comment]
LGTM now :+1:

**Comment by antonjarl** [comment]
Nice work Emil!

**Comment by dbackeus** [comment]
Would go for completing the close-button as well.

Maybe a pure css solution can be done using `:before` + `content:` rules?

Can @eliashedlund comment on the status of the buttons?

**Pull request merged**